libraries/AP_ICEngine/AP_ICEngine.cpp

```diff
@@ -167,6 +167,11 @@ const AP_Param::GroupInfo AP_ICEngine::var_info[] = {
 
     // 18 was IGNITION_RLY
 
+#if AP_ICENGINE_TCA9554_STARTER_ENABLED
+    // @Group: _TCA9554
+    // @Path: AP_ICEngine_TCA9554.cpp
+    AP_SUBGROUPINFO(tca9554_starter, "_TCA", 19, AP_ICEngine, AP_ICEngine_TCA9554),
```

Comment:
maybe just a ICE_OPTIONS bit instead? That would be a lot smaller patch and not clutter up parameters. We only need a single bit for the direction

Reply:
yes that would be good, as if there where other implementation of cranking they can use that flag too for direction control.

Reply:
@tridge I have made the changes, have a look.

Also, if there is a need to splitting the commit into one for param mods and other for implementation then let me know.